src/App.js

```diff
@@ -0,0 +1,29 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit
+          {' '}
+          <code>src/App.js</code>
+          {' '}
+          and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
```

Comment:
When I try to run your app, I receive a compile error.

![image](https://user-images.githubusercontent.com/38088818/177182713-7797a391-bb64-4d24-8c9f-3968a87802c0.png)

This is because you need to import react with Component. This means you also need to refactor your component into being class-based.
```
import React, { Component }  from 'react';
```
- Kindly refactor `App.js` to the way it was, as a class-based component. For now, you will be using classes to create components. You then refactor to functional components in future projects.
i.e
```
class App extends Component {
  render () {
    return (
      <div className="App">
         ...
      </div>
    )
  };
}
```